.github/CHANGELOG.md

```diff
@@ -424,10 +457,10 @@ and requirements-ci.txt (unpinned). This latter would be used by the CI.
 
```

Comment:
I can't comment, but I would remove the 'separate requirements file for the CI issue' one, since that was a change to the repo continuous integration, rather than the core PL package itself.

Reply:
Done.